docs/ephemerisgen.rst

```diff
@@ -115,7 +115,11 @@ If you want to use the same input orbits across multiple Sorcha runs, you can sa
 
 .. attention::
    Currently the Sorcha-generated ephemeris is outputted in CSV, whitespace or HDF5 file format only.
+   
 
+.. attention::
```

Comment:
This needs to go out the output page. Not here. This is about inputting ones own ephemeris, so it seems confusing for this go here. I think it goes in the output where we say that the output is formatted in X order if an ephemeris file is provided. Can you move this